packages/Breadcrumb/src/breadcrumb.scss

```diff
@@ -12,13 +15,40 @@
 
   /* Link */
   --ids-breadcrumb-link-color: #{tokens.$grey-600};
-  --ids-breadcrumb-link-border: .1rem solid #{tokens.$grey-600};
+  --ids-breadcrumb-link-color-hover: #{tokens.$grey-600};
+  --ids-breadcrumb-link-border: .1rem solid transparent;
+  --ids-breadcrumb-link-border-color-hover: #{tokens.$grey-600};
+  --ids-breadcrumb-link-text-decoration-hover: none;
+  --ids-breadcrumb-link-focus: #{tokens.$focus};
+  --ids-breadcrumb-link-border-radius-focus: #{tokens.$border-radius-xs};
 
   /* Chevron */
   --ids-breadcrumb-chevron-color: #{tokens.$grey-800};
   --ids-breadcrumb-chevron-spacing: 0 #{tokens.$space-2};
 }
 
+[data-brand="workleap"] {
+  /* Default */
+  --ids-breadcrumb-font-family: var(--hop-overline-font-family);
+  --ids-breadcrumb-font-size: var(--hop-overline-font-size);
+  --ids-breadcrumb-line-height: var(--hop-overline-line-height);
+
+  /* Item */
+  --ids-breadcrumb-item-color: var(--hop-neutral-text);
+
+  /* Link */
+  --ids-breadcrumb-link-color: var(--hop-neutral-text-weak);
+  --ids-breadcrumb-link-color-hover: #3c3c3c; /* Will be var(--hop-neutral-text-weak-hover) */
```

Comment:
I would use neutral-text in the meantime. As hardcoded colours won't react to theming, and won't be updated when tokens update.